kong/cmd/utils/migrations.lua

```diff
@@ -87,38 +75,39 @@ local function up(schema_state, db, opts)
       -- legacy: migration from 0.14 to 1.0 cannot be performed
       if schema_state.legacy_missing_component then
         error(fmt("Migration to 1.0 can only be performed from a 0.14 %s " ..
-                  "%s, but the current one seems to be older (missing "    ..
-                  "migrations for '%s'). Migrate to 0.14 first, or "  ..
+                  "%s, but the current %s seems to be older (missing "     ..
+                  "migrations for '%s'). Migrate to 0.14 first, or "       ..
                   "install 1.0 on a fresh %s.", db.strategy, db.infos.db_desc,
-                  schema_state.legacy_missing_component, db.infos.db_desc))
+                  db.infos.db_desc, schema_state.legacy_missing_component,
+                  db.infos.db_desc), 2)
       end
 
       if schema_state.legacy_missing_migration then
         error(fmt("Migration to 1.0 can only be performed from a 0.14 %s " ..
-                  "%s, but the current one seems to be older (missing "    ..
+                  "%s, but the current %s seems to be older (missing "     ..
                   "migration '%s' for '%s'). Migrate to 0.14 first, or "   ..
                   "install 1.0 on a fresh %s.", db.strategy, db.infos.db_desc,
-                  schema_state.legacy_missing_migration,
-                  schema_state.legacy_missing_component, db.infos.db_desc))
+                  db.infos.db_desc, schema_state.legacy_missing_migration,
+                  schema_state.legacy_missing_component, db.infos.db_desc), 2)
       end
 
       error(fmt("Migration to 1.0 can only be performed from a 0.14 %s " ..
-                "%s, but the current one seems to be older (missing "    ..
+                "%s, but the current %s seems to be older (missing "     ..
                 "migrations). Migrate to 0.14 first, or install 1.0 "    ..
                 "on a fresh %s.", db.strategy, db.infos.db_desc,
-                db.infos.db_desc))
+                db.infos.db_desc, db.infos.db_desc), 2)
     end
 
     if schema_state.legacy_is_014 then
       local present, err = db:are_014_apis_present()
       if err then
-        error(err)
+        error(err, 2)
```

Comment:
Why are we increasing the stack level in those errors? In our CLI code, we never do so, as it allows us to accurately pinpoint where they were thrown. The trace doesn't appear for users, only when `--vv` is specified (and we are the ones usually having a look and we want to know where it was thrown from, precisely. Especially in such cases where the error isn't prefixed with any sort of static strings.